js/preset.js

```diff
@@ -26,89 +25,115 @@
-    sliderPresetLevel.value = value;
+    var value = (sliderPin.offsetLeft * MAX_BLUR
+                          / sliderLineWidth).toFixed(2);
     return value;
   };
 
   var getBrightness = function () {
-    var value = ((sliderPin.offsetLeft * BRIGHTNESS_VALUE)
-                          / sliderLine.clientWidth
-                          + BRIGHTNESS_CORRECTION).toFixed(2);
-    sliderPresetLevel.value = value;
+    var value = (((sliderPin.offsetLeft
+                          * (MAX_BRIGHTNESS - BRIGHTNESS_CORRECTION))
+                          / sliderLineWidth) + BRIGHTNESS_CORRECTION).toFixed(2);
     return value;
   };
 
   // change filter saturation --------------------------------------------------
   var setPresetValue = function () {
-    var preset = document.querySelector('.effects');
-    var presetChrome = preset.querySelector('#effect-chrome');
-    var presetSepia = preset.querySelector('#effect-sepia');
-    var presetMarvin = preset.querySelector('#effect-marvin');
-    var presetPhobos = preset.querySelector('#effect-phobos');
-    var presetHeat = preset.querySelector('#effect-heat');
-
-    if (presetChrome.checked) {
-      picturePreview.style.filter = 'grayscale(' + getSaturation() / 100 + ')';
-    }
-    if (presetSepia.checked) {
-      picturePreview.style.filter = 'sepia(' + getSaturation() / 100 + ')';
-    }
-    if (presetMarvin.checked) {
-      picturePreview.style.filter = 'invert(' + getSaturation() + '%' + ')';
-    }
-    if (presetPhobos.checked) {
-      picturePreview.style.filter = 'blur(' + getBlur() + 'px' + ')';
-    }
-    if (presetHeat.checked) {
-      picturePreview.style.filter = 'brightness(' + getBrightness() + ')';
+    var filter = document.querySelector('#upload-select-image input[name="effect"]:checked');
+    var presetResult;
+
+    switch (filter.value) {
+      case 'chrome':
+        presetResult = getSaturation();
+        picturePreview.style.filter = 'grayscale(' + presetResult + ')';
+        break;
+      case 'sepia':
+        presetResult = getSaturation();
+        picturePreview.style.filter = 'sepia(' + presetResult + ')';
+        break;
+      case 'marvin':
+        presetResult = getSaturation() * 100;
+        picturePreview.style.filter = 'invert(' + presetResult + '%' + ')';
+        break;
+      case 'phobos':
+        presetResult = getBlur();
+        picturePreview.style.filter = 'blur(' + presetResult + 'px' + ')';
+        break;
+      case 'heat':
+        presetResult = getBrightness();
+        picturePreview.style.filter = 'brightness(' + presetResult + ')';
+        break;
     }
+    sliderPresetLevel.value = presetResult;
   };
 
   // filter select from filter list ---------------------------------------------
   var onPresetClick = function (evt) {
     var target = evt.target;
+    sliderLineWidth = sliderLine.offsetWidth;
     resetPreset();
     showSlider();
     if (target.value === 'none') {
```

Comment:
можно еще показывать/скрывать слайдер с помощью одной функции
```suggestion
   toggleSlider(target.value !== 'none')
```

где 
```
var toggleSlider= function (isVisible) {
    sliderBlock.classList.toggle('hidden', isVisible);
};
```